$OpenBSD: patch-unix_installManPage,v 1.3 2013/11/05 04:45:18 stu Exp $

Put macros after .TH to prevent them from being redefined. Tcl bug 426679ef7d.

--- unix/installManPage.orig	Fri Sep 13 12:07:56 2013
+++ unix/installManPage	Mon Nov  4 11:06:19 2013
@@ -103,7 +103,7 @@ for Target in $Names; do
     rm -f $Dir/$Target $Dir/$Target.*
     if test -z "$First" ; then
 	First=$Target
-	sed -e "/man\.macros/r $SrcDir/man.macros" -e "/man\.macros/d" \
+	sed -e "/man\.macros/d" -e "/\.TH/r $SrcDir/man.macros" \
 	    $ManPage > $Dir/$First
 	chmod 444 $Dir/$First
 	$Gzip $Dir/$First
